tests/ert/unit_tests/config/test_forward_model.py

```diff
@@ -1021,7 +1047,9 @@ def __init__(self):
                 command=["the_executable.sh"],
             )
 
-        def validate_pre_experiment(self, fm_step_json: ForwardModelStepJSON) -> None:
+        def validate_pre_experiment(
+            self, fm_step_json: ForwardModelStepJSON, _: dict
```

Comment:
Very nitpick, but could this be Dict[str, str] instead of dict ?

Reply:
fixed.